status/versions.go

```diff
@@ -80,16 +81,36 @@ func tracingVersion(conf *config.Config, homeClusterSAClient kubernetes.ClientIn
 	product.Name = string(tracingConfig.Provider)
 	product.Url = tracingConfig.URL
 
-	if product.Url != "" {
+	// we want to go to inClusterURL to obtain the version. If it isn't specified, fallback to the external URL.
+	versionUrl := tracingConfig.InClusterURL
+	if versionUrl == "" {
+		versionUrl = tracingConfig.URL
+	}
+
+	// there is no known way to get the version from GRPC. So we'll try to change the URL to go over HTTP,
+	// but this is not guaranteed to work. But it is worth a try.
+	if tracingConfig.UseGRPC {
+		parsedUrl, err := url.Parse(versionUrl)
+		if err == nil {
+			// strip the port - if the URL is http, it'll go over 80, if https, it'll go over 443
+			if host := parsedUrl.Hostname(); host != "" {
```

Comment:
I would strip the port just for Jaeger, because Tempo is going to set the URL for the http client with the port also when using grpc (And then it wouldn't make sense to remove the port, IMO). Also, the Tempo operator has an option to setup grpc in the same port as http, `stream_over_http_enabled`, so http and GRPC would be using the same port and the version would be available. 
For example, in this configuration: 
```
      tracing:
        enabled: true
        grpc_port: 9095
        in_cluster_url: http://tempo-cr-query-frontend.tempo:3200
        provider: tempo
        url: 'http://tempo-cr-query-frontend.tempo:3200'
        use_grpc: true
```
With this is not possible to get the version: 
![image](https://github.com/user-attachments/assets/9bf5eeb5-4175-4c37-9ffd-7c7803c3a673)

But the code that we have now makes it possible: 
![image](https://github.com/user-attachments/assets/e30f3753-4237-4f4f-acb4-e4ba6a260ef7)

Reply:
OK, check the latest commit. I just moved that block of code down into the "if-jaeger" section.